uptane-standard.md

```diff
@@ -716,7 +716,7 @@ The Primary SHOULD send the time to each ECU.
 
 #### Send metadata to Secondaries {#send_metadata_primary}
 
-The Primary SHALL send its latest downloaded metadata to all of its associated Secondaries. The metadata it sends to each Secondary SHALL include all of the metadata required for verification on that Secondary. For full verification Secondaries, this includes the metadata for all four roles from both repositories, plus any delegated Targets metadata files the Secondary will recurse through to find the proper delegation. For partial verification Secondaries, this could include fewer metadata files; at a minimum, it includes only the Targets metadata file from the Director repository.
+The Primary SHALL make available to each of its associated Secondaries all of the metadata required for verification on that Secondary. For full verification Secondaries, this includes the metadata for all four roles from both repositories, plus any delegated Targets metadata files the Secondary will recurse through to find the proper delegation. For partial verification Secondaries, this could include fewer metadata files; at a minimum, it includes only the Targets metadata file from the Director repository.
```

Comment:
This also removes the word "latest". Perhaps not critical, but maybe "... all of the latest metadata required for verification" would be better.

Reply:
Multiple versions of Root metadata may be required for verification on a full verification Secondary, e.g. if the Root metadata was rotated multiple times since the vehicle's last connectivity. I worry that the word "latest" could imply that only the latest single version needs to be made available to Secondaries.

Reply:
Another scenario to consider. Vehicle has one Primary and one Secondary ECU.
1. Primary and Secondary currently have Director Root metadata v1.
1. Primary retrieves Director Targets metadata v8.
1. As part of verifying Director Targets metadata v8, Primary discovers and rotates to Director Root metadata v2.
1. Primary verifies Director Targets metadata v8
1. The Primary notices that there is no target specified for the Secondary. As discussed in https://github.com/uptane/deployment-considerations/issues/134, the Primary may choose to skip sending the Targets metadata to the Secondary. Since there is nothing the Secondary needs to verify, it doesn't urgently need the Director Root metadata v2. Using the terminology in the standard, the Director Root metadata v2 could be classified as _not_ "required for verification on that Secondary" in this scenario. This classification is up for debate, but it may be desirable for some implementations. E.g. an implementation may not wish to wait for a Secondary to wake up to rotate Root metadata when there is no update for that Secondary.
1. Later, Primary retrieves Director Targets metadata v9.
1. As part of verifying Director Targets metadata v9, Primary discovers and rotates to Director Root metadata v3.
1. Director Targets metadata v9 includes a target for the Secondary. The following metadata is now required for verification on the Secondary:
    - Director Root metadata v2
    - Director Root metadata v3
    - Director Targets metadata v9

In this scenario the Secondary requires Director Root metadata v2, which is neither the latest version of the metadata nor part of the latest set of metadata downloaded by the Primary. Depending on the implementation, the primary may serve Root metadata v2 from a cache or redownload it on request.